packages/cli/src/loaders/xliff.ts

```diff
@@ -0,0 +1,17 @@
+const xliff2js: any = require('xliff/xliff2js');
+const js2xliff: any = require('xliff/js2xliff');
```

Comment:
damn this is brutal

is there a way to import those as usually, by any chance?